android/engine/src/test/assets/configs/app/implementation_guide_config.json

```diff
@@ -0,0 +1,55 @@
+{
+  "resourceType": "ImplementationGuide",
+  "id": "12967310",
+  "url": "ImplementationGuide/quest",
+  "version": "1.1.0",
+  "name": "quest",
+  "title": "Quest Implementation Guide",
+  "status": "draft",
+  "packageId": "org.smartregister.fhircore",
+  "date": "2024-04-01",
+  "publisher": "Ona Systems, Inc.",
+  "fhirVersion": [
+    {
+      "code": {
+        "coding": [
+          {
+            "system": "http://hl7.org/fhir/FHIR-version",
+            "code": "4.3.0",
+            "display": "4.3.0"
+          }
+        ],
+        "text": "FHIR Release 4B."
+      }
+    }
+  ],
+  "useContext": {
+    "code": {
+      "coding": [
+        {
+          "system": "http://terminology.hl7.org/CodeSystem/usage-context-type",
+          "code": "program",
+          "display": "Program"
+        }
+      ],
+      "text": "Program"
+    },
+    "valueRange": {
+      "low": {
+        "value": 1
+      },
+      "high": {
+        "value": 10
+      }
+    }
+  },
+  "definition": {
+    "resource": [
+      {
+        "reference": {
+          "reference": "Composition/8294"
+        }
+      }
```

Comment:
embed the entire composition content here

Reply:
@pld Do you mean transfer the list of references in the `Composition.section` field to `ImplementationGuide.definition.resource` or embed the entire composition resource to the IG?

The IG's definition.resource expects references as the links to the resources. It is not able to take the full resource as we do in `QR.contained`

Reply:
OK nevermind then, I got the structure of the IG wrong, looks good